vital/algo/compute_ref_homography.h

```diff
@@ -62,12 +65,16 @@ class VITAL_ALGO_EXPORT compute_ref_homography
 
 protected:
   compute_ref_homography();
-
 };
 
-/// Shared pointer type of base compute_ref_homography algorithm definition class
-typedef std::shared_ptr<compute_ref_homography> compute_ref_homography_sptr;
+/// Shared pointer type of base compute_ref_homography algorithm definition
+/// class
+typedef std::shared_ptr< compute_ref_homography > compute_ref_homography_sptr;
+
+} // namespace algo
+
+} // namespace vital
 
-} } } // end namespace
+}     // end namespace
```

Comment:
can we fix the spacing before the comment and change the comment to // namespace kwiver